caom2repo/caom2repo/core.py

```diff
@@ -305,15 +304,15 @@ def process_observation_id(self, collection, observation_id,
             observation = self.get_observation(collection, observation_id)
             orig_checksum = observation.acc_meta_checksum
             if orig_checksum:
-                orig_checksum = orig_checksum.uri
+                orig_checksum = orig_checksum
             if self.plugin.update(observation=observation,
                                   subject=self._subject) is False:
-                self.logger.info('SKIP {}'.format(observation.observation_id))
-                skipped = observation.observation_id
+                self.logger.info('SKIP {}'.format(observation_id))
```

Comment:
You've probably already considered this, but should this method use `observation_id` or `uri`?

Reply:
I've left this unchanged on purpose because I'm not sure how the service API is going to look like. Pat was not sure if it could work with observation URIs. So this is technical debt that we need to review once the repo service is available.